Lib/test/test__colorize.py

```diff
@@ -166,6 +166,17 @@ def test_colorized_detection_checks_for_file(self):
                 file.isatty.return_value = False
                 self.assertEqual(_colorize.can_colorize(file=file), False)
 
+            # The documentation for file.fileno says:
+            # > An OSError is raised if the IO object does not use a file descriptor.
+            # See https://github.com/python/cpython/issues/141570
+            with unittest.mock.patch("os.isatty", side_effect=ZeroDivisionError):
```

Comment:
why does this need the ZeroDivisionError side effect?

Reply:
It copies the previous block. I suppose it's to verify it never gets called.